modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-grid/src/main/resources/META-INF/resources/grid_field.js

```diff
@@ -121,24 +100,20 @@ AUI.add(
 						container.all('.help-block').appendTo(container.one('.form-group'));
 					},
 
-					_getGridRowsNode: function() {
+					_getFocusTarget: function() {
 						var instance = this;
 
-						var container = instance.get('container');
-
-						var gridRowsNode = container.all('tbody tr');
-
-						return gridRowsNode;
+						return instance.focusTarget;
```

Comment:
Since the `focusTarget` is a private element it should star with a underscore `_ focusTarget`